src/elements/content-sidebar/activity-feed/common/activity-message/TruncatableMessage.js

```diff
@@ -0,0 +1,39 @@
+// @flow
+import * as React from 'react';
+import classNames from 'classnames';
+import ToggleMoreLessButton from './ToggleMoreLessButton';
+import './ActivityMessage.scss';
+
+type Props = {
+    children: React.Node,
+};
+
+export default function TruncatableMessage({ children }: Props) {
+    const [isTruncated, setTruncated] = React.useState(true);
+    const [shouldTruncate, setShouldTrucate] = React.useState(false);
+    const messageContainer = React.useRef<HTMLDivElement | null>(null);
+
+    React.useLayoutEffect(() => {
+        if (messageContainer.current) {
+            const { clientHeight, scrollHeight } = messageContainer.current;
+            setShouldTrucate(clientHeight !== scrollHeight || clientHeight > 140);
```

Comment:
We're currently duplicating this value in the stylesheet. Can we share the value between the component and stylesheet? If not, should we apply this one style rule via inline styles, instead?